test/geometry_tests.cpp

```diff
@@ -162,6 +204,48 @@ TEST(Point2DPlanning_Griz_Test, bresenham_conversion_octant1_test) {
 
 }
 
+TEST(Point2DPlanning_Griz_Test, raytraceLine_conversion_pointTest) {
+	Cell startPix = {.x = 0, .y = 0};
+	Cell endPix = {.x = 0, .y = 0};
+	auto pixels = raytraceLine_conversion(startPix, endPix);
```

Comment:
By default, make everything `const` unless you can't

Reply:
Done

Reply:
Done, I think